<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MT500</class>
 <widget class="QMainWindow" name="MT500">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MT500</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="DataTab">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QTextBrowser" name="DataBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>551</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clrButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>330</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cloud</string>
     </attribute>
     <widget class="QTextBrowser" name="CloudBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>551</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="StatusTab">
     <attribute name="title">
      <string>Status</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>551</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="gidLabel">
         <property name="text">
          <string>Locality GID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="inCOMLabel">
         <property name="text">
          <string>Incoming COM Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outCOMLabel">
         <property name="text">
          <string>Outgoing COM Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rxLabel">
         <property name="text">
          <string>Messages Received:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="reconfigureButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Update Configuration</string>
       </property>
      </widget>
      <widget class="QLabel" name="updateLbl">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>163</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Run Mode:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="runModeComboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>155</y>
         <width>78</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
      </widget>
     </widget>
     <zorder>frame_4</zorder>
     <zorder>verticalLayoutWidget</zorder>
    </widget>
    <widget class="QWidget" name="PortsTab">
     <attribute name="title">
      <string>Ports</string>
     </attribute>
     <widget class="QTableWidget" name="ipTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>431</width>
        <height>211</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>411</width>
        <height>0</height>
       </size>
      </property>
      <row>
       <property name="text">
        <string>Destination 1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Destination 2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Destination 3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Destination 4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Destination 5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Destination 6</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>IP Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Port</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data Type</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="SetupTab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add a Port</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>IP Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ipLineEdit">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portLineEdit">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Data Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataSelect">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Raw</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Processed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Base Station</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="addButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>70</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QLabel" name="addLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>581</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="addLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Delete a Port</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>391</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>IP Address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="delSelect">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="delLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>581</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Test Connection</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="testBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>561</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="testButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>10</y>
         <width>86</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>frame</zorder>
     <zorder>label_8</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>frame_3</zorder>
    </widget>
    <widget class="QWidget" name="AboutTab">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MT500 is a product of MT Products Division of</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>60</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MapTech, Inc. All Rights Reserved.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>120</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MapTech, Inc.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>150</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3154 State St.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>180</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Blacksburg, VA 24060</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>210</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>540-961-7864</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>240</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>www.maptech-inc.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="versionLbl">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>290</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Program Version: V.007</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
